engine/src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -25,10 +27,25 @@ import scala.sys.process._
 import scala.util.{Failure, Success, Try}
 
 object SgeBackend {
+
+implicit class RichConfig(val underlying: Config) extends AnyVal {
+  def getOptionalString(path: String): Option[String] = {
+    try {
+     return Some(underlying.getString(path))
+    } catch  {
+     case e: Exception => {
+	None
+     } 
+    } 
+   } 
+}
```

Comment:
There's existing functionality in the Cromwell-derived [Lenthall](https://github.com/broadinstitute/lenthall/blob/develop/src/main/scala/lenthall/config/ScalaConfig.scala) library to get `Option[String]` config values.